### PR TITLE
Add IServiceSelector to exported index file

### PR DESCRIPTION
**Pull request by janhalama**
## Description

This PR adds IServiceSelector to exported index file.

## Motivation and Context

We need to work with IServiceSelector in @superfaceai/testing library.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have updated the documentation accordingly. If you are changing **code related to user secrets** you need to really make sure that [security documentation](https://github.com/superfaceai/one-sdk-js/blob/main/SECURITY.md) is correct.
- [x] I have read the [CONTRIBUTING](https://github.com/superfaceai/one-sdk-js/blob/main/CONTRIBUTING.md) document.
- [x] I haven't repeated the code. (DRY)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by lukas-valenta** [comment]
I'm not sure we want to make stuff for testing library public anymore. What about creating a `src/private/index.ts`, exporting it from there and importing from `@superfaceai/one-sdk/private` in testing lib?

**Comment by janhalama** [comment]
> I'm not sure we want to make stuff for testing library public anymore. What about creating a `src/private/index.ts`, exporting it from there and importing from `@superfaceai/one-sdk/private` in testing lib?

I just have to figure out how to configure multiple exports.

**Comment by lukas-valenta** [comment]
it's okay to import `@superfaceai/one-sdk/dist/private` if it would be difficult to configure

File: src/index.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ export * from './client';
 export * from './interfaces';
 export * from './lib';
 export * from './internal';
+export * from './private';
```
Comment (lukas-valenta) [inline src/index.ts:8]: don't export this

**Review by lukas-valenta — CHANGES_REQUESTED** [review]

**Review by lukas-valenta — APPROVED** [review]

**Pull request merged**